azure-pipelines.yml

```diff
@@ -59,6 +67,106 @@ jobs:
       projects: '**/*UnitTest/*.csproj'
       arguments: '--configuration $(buildConfiguration)'
 
+  - task: DotNetCoreCLI@2
+    displayName: 'E2E tests for Spark 2.3.0 with oldest compatible worker $(oldestCompatibleWorker)'
+    inputs:
+      command: test
+      projects: '**/Microsoft.Spark.E2ETest/*.csproj'
+      arguments: '--configuration $(buildConfiguration) --filter $(TestsToFilterOut)'
+    env:
+      SPARK_HOME: $(Build.BinariesDirectory)\spark-2.3.0-bin-hadoop2.7
+      DOTNET_WORKER_DIR: $(DotnetWorkerDir)
```

Comment:
are we running against the new worker?

Reply:
oh sorry my bad, changing back to old worker, using a different variable for that to avoid confusion